service/worker/scanner/shardscanner/activities.go

```diff
@@ -383,39 +381,3 @@ func fixShard(
 	}
 	return &report, nil
 }
-
-// ScannerEmitMetricsActivity will emit metrics for a complete run of ShardScanner
-func ScannerEmitMetricsActivity(
```

Comment:
Thanks for the clean-up @mantas-sidlauskas! Overall, I like it but I think we should keep **ShardSuccessCount** and **ShardControlFlowFailureCount** so that we can keep an eye on the health of the scanner and perhaps create alerts for.  I'd suggest either keeping these as aggregated metrics or emitting from the scan activity just like you did in your PR #4476. What do you think?